fix: "Cannot read properties of undefined (reading '__esModule')" Error

Swastik19Nit opened the pull request:
# Fix "Cannot read properties of undefined (reading '__esModule')" Error

## Issue  (Solves #591 )
The application encountered the following runtime error:  

### **Cause**  
1. **Direct Access to Vuex State**  
   - The original implementation accessed `this.$store.state.error` directly in the `error()` method.  
   - When the state was uninitialized or reset, `errorCode` and `message` were `undefined`, causing the error.

2. **Improper Toast Invocation**  
   - The Vue Toastification instance (`this.$toast.error`) was used without proper initialization.  

### **Fix**  
- **Used `mapState` to Bind State Correctly**  
  ```js
  computed: {
    ...mapState({
      error: (state) => state.error,
    }),
  }

Swastik19Nit commented:
@marcgc21 No more errors in console.
![image](https://github.com/user-attachments/assets/2d119ffb-2372-4c6c-b987-9cbc3ea3fdc3)

jvJUCA reviewed (changes requested):
For me, these changes seem to have broken the GlobalToast component.

![image](https://github.com/user-attachments/assets/7e5a4aff-3ddc-4f81-a5a5-3aa0e635f9f4)

Swastik19Nit commented:
Removed the created hook since we don't need to initialize anything
Changed this.toast.error to this.$toast.error to use the globally registered toast
@jvJUCA  I guess it should solve the error. Kindly let me know any changes if needed.

Swastik19Nit commented:
@jvJUCA Could you please kindly review it again?

jvJUCA reviewed (approved):
Thanks for the fix, @Swastik19Nit! I really appreciate it—now the console errors are completely gone!
